zeppelindemo/resources/projects.xml

```diff
@@ -76,18 +76,71 @@ into the zeppelin-env.sh file. <br />
 <pre>
 $ sudo /opt/zeppelin/bin/zeppelin-daemon.sh start
 </pre>
-6. Configure Cassandra log appenders.<br />
+6. Downgrade Cassandra version, since Zeppelin compatible only with not higher than 2.2 (by default on Sandbox Cassandra version 3.5 is installed). <br />
```

Comment:
Please avoid exact values in the documentation when possible ("by default on Sandbox Cassandra version 3.5 is installed" can be re-written in the form of "by default Sandbox uses newer version").